### ISSUE TITLE
problem after deploying branch with "/" in the name

### ISSUE THREAD

**Issue by mathroc** [ezweb/git-up#9]
eg:

```
mathieu on project:master $ git up preprod origin/master
>>> Connect to remote git@host ...
|--------------------------------------------------------------------|
| fl-api <preprod> is on feature                      |
| >> deploying origin/master <<                   |
|--------------------------------------------------------------------|
[preprod] DIFF:  feature..origin/master
fatal: ambiguous argument 'feature..origin/master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

**Comment by gregorg** [comment]
Fixed with commit 836c057

**Comment by mathroc** [comment]
thx!
